crowbar_framework/app/models/api/node.rb

```diff
@@ -16,16 +16,20 @@
 
 module Api
   class Node < Tableless
+    attr_reader :node_name
```

Comment:
But you're right when moving to ActiveRecord the column name will be name I guess so it makes sense to change it to name!